### ISSUE TITLE
Running FASTQC on imported samples is super slow

### ISSUE THREAD

**Issue by glebkuznetsov** [churchlab/millstone#629]
For ~100 samples (Fix Recoli data), to copy the data into the millstone filesystem and then run FastQC takes about 250 sec.

Maybe we can make running FastQC optional?

Is there any other way to make this faster?

**Comment by dbgoodman** [comment]
Currently, it should run in the background so that you can still continue
with other operations. I also have no problem making it optional.

**Comment by glebkuznetsov** [comment]
I think we should make this optional on a per sample basis (via the dropdown menu)

**Comment by dbgoodman** [comment]
agreed. the corresponding wishlist feature would be aggregation of the
FASTQC output in the millstone table view - how many reads per sample, how
many adapters are present, what the average read quality is, any red flags,
etc
